tool_for_bioinformatics.py

```diff
@@ -0,0 +1,153 @@
+from typing import Union
+import os
+import modules.nucleic_acids_functions as na
+import modules.fastq_filters as ff
+import modules.amino_acids_functions as aa
+NUCLEOTIDES = {'U', 'A', 'g', 't', 'G', 'T', 'a', 'c', 'C', 'u'}

```

Comment:
```suggestion
import os
from typing import Union

import modules.nucleic_acids_functions as na
import modules.fastq_filters as ff
import modules.amino_acids_functions as aa


NUCLEOTIDES = {'U', 'A', 'g', 't', 'G', 'T', 'a', 'c', 'C', 'u'}
```

С импортами тоже куча своих конвенций. Например тут:
1) Отделяем от дальнейшего кода 2 строками
2) Сперва внешние, потом твои
3) Сперва import, потом from ... import 

Ну и там далее еще куча разных, можно про это в PEP-8 прочитать